CHANGES.rst

```diff
@@ -14,6 +14,13 @@ documentation
 
 - Update tutorial usage of ``jump.threshold`` to ``jump.rejection_threshold``. [#7572]
 
+
+extract_1d
+----------
+-For NIRSpec IFU point source data remove options assuming data is in flux units. Treat these data
```

Comment:
Need a whitespace between the dash and the first character of text. So the 2nd line needs to be intended one more space too.